src/hotspot/share/adlc/forms.cpp

```diff
@@ -268,6 +268,7 @@ Form::DataType Form::is_load_from_memory(const char *opType) const {
   if( strcmp(opType,"LoadRange")==0 )  return Form::idealI;
   if( strcmp(opType,"LoadS")==0 )  return Form::idealS;
   if( strcmp(opType,"LoadVector")==0 )  return Form::idealV;
+  if( strcmp(opType,"VectorMaskedLoad")==0 )  return Form::idealV;
```

Comment:
More of a bike shedding question:
The patterns is LoadRange, LoadS, LoadVector - why not name it in the same style - LoadVectorMasked?